tests/unit/helpers.js

```diff
@@ -235,3 +236,11 @@ export function createFakeEvent(extraProps = {}) {
     ...extraProps,
   };
 }
+
+export function createStubErrorHandler(capturedError = null) {
+  return new ErrorHandler({
+    id: 'some-id',
```

Comment:
Might be nice to make this something like `create-stub-error-handler-id` so it's obvious where it came from.